.circleci/config.yml

```diff
@@ -359,6 +359,35 @@ jobs:
       - store_artifacts:
           path: /go/src/github.com/tendermint/tendermint/tendermint-*.tar.gz
 
+  # Test RPC implementation against the swagger documented specs
+  contract_tests:
+    working_directory: /home/circleci/.go_workspace/src/github.com/tendermint/tendermint
+    machine:
+      image: circleci/classic:latest
```

Comment:
why not default one or nodejs?

Reply:
For this @sabau mentioned its either start in node and download go or start in go and download node, he chose node.

Reply:
he chose neither!!